src/db12/factors.json

```diff
@@ -5,6 +5,12 @@
 				"Intel": 0.86,
 				"AMD": 0.71
 			}
+		},
+		"3.11": {
+			"cpu_brand": {
+				"Intel": 0.912400392678881,
+				"AMD": 0.912400392678881
+			}
```

Comment:
If I understand correctly, these factors aim at aligning the DB12 score, computed with python3.11, with the HEPScore. Am I right?  

`factors.json` was originally introduced to align the original DB12 score, computed with python2, with the DB12 score computed with python3.9.

The questions are:
- do we want to introduce this HEPScore scaling factor directly in the DB12 repo?
  - DB12 can also be used outside of the LHC context
  - The HEPSPEC06 scaling factor was set in LHCbDIRAC
  
- do we still want to be aligned with the original DB12 score (python2)?
  - DB12 can be used outside of the LHC context again. 
  - Regarding LHCb MC sim applications, the raw python3.9 DB12 score was more accurate than the original DB12 score on AMD CPUs (and a bit less accurate on Intel but not significantly...)
  - May be we need to compare the raw python3.11 DB12 score to the LHCb MC sim applications now.

Any opinion about that @fstagni?